vpr/src/place/place.cpp

```diff
@@ -1463,10 +1494,23 @@ static e_move_result try_swap(float t,
             delta_c = bb_delta_c;
         }
 
+        if (manual_move) {
+            //update all the costs in the manual_move_info variable and open cost summary window
+            manual_move_info->delta_c = delta_c;
+            manual_move_info->bb_delta_c = bb_delta_c;
+            manual_move_info->timing_delta_c = timing_delta_c;
+            cost_summary_window();
+            update_screen(ScreenUpdatePriority::MAJOR, " ", PLACEMENT, nullptr);
+        }
+
         /* 1 -> move accepted, 0 -> rejected. */
         move_outcome = assess_swap(delta_c, t);
```

Comment:
If you want the cost summary window to display the suggested move outcome, you should move the ```assess_swap``` routine before ```cost_sumary_window()```.

Reply:
good point. resolved